src/plugins/intel_cpu/src/ngraph_transformations/convert_to_interaction.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "convert_to_interaction.hpp"
+#include "op/interaction.hpp"
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <transformations/utils/utils.hpp>
+
+#include "itt.hpp"
+
+ov::intel_cpu::ConvertToInteraction::ConvertToInteraction() {
+    MATCHER_SCOPE(ConvertToInteraction);
+    using namespace ngraph::pattern;
+    auto dense_feature_m = any_input(has_static_rank());
+    std::vector<std::shared_ptr<Node>> features_m{dense_feature_m};
+    OutputVector features_output{dense_feature_m->output(0)};
+    const int sparse_feature_num = 26;
+    for (size_t i = 0; i < sparse_feature_num; i++) {
+        auto feature = any_input(has_static_rank());
+        features_m.push_back(feature);
+        features_output.push_back(feature->output(0));
+    }
+    auto concat_m = wrap_type<ngraph::opset8::Concat>(features_output);
+    auto reshape_m = wrap_type<ngraph::opset8::Reshape>({concat_m->output(0), any_input()->output(0)});
+    // This transpose is moved due to TransposeMatmul Transformation
+    // auto transpose_m = wrap_type<ngraph::opset8::Transpose>({reshape_m->output(0), any_input()->output(0)});
```

Comment:
Please remove or uncomment.

Reply:
Sure, I will remove this since the transpose op is moved by **TansposeMatmul** Transformation